### PR TITLE
2022 2.0

### PR DESCRIPTION
**Pull request by gwbischof**
```diff
diff 2022-2.0-py37-env.yml ~/dama/conda-pack-template/envs/nsls2-collection-2022-1.0-py37-env.yml
1c1
< name: 2022-2.0-py37
---
> name: nsls2-collection-2022-1.0-py37
18c18
<   - bluesky-kafka >=0.9.0  # [not win]
---
>   - bluesky-kafka >=0.8.0  # [not win]
21d20
<   - bluesky-queueserver-api
29c28
<   - databroker >=1.2.5,<2
---
>   - databroker >=1.2.0
76c75
<   - nslsii >=0.4.0
---
>   - nslsii >=0.3.2
83c82
<   - ophyd >=1.6.4
---
>   - ophyd >=1.6.3
113c112
<   - silx >=0.15.2
---
>   - silx >=0.15.2,<1
```

xref https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/1

### CODE REVIEW

**Comment by gwbischof** [comment]
```diff
diff 2022-2.0-py39-env.yml ~/dama/conda-pack-template/envs/nsls2-collection-2022-1.0-py39-env.yml
1c1
< name: 2022-2.0-py39
---
> name: nsls2-collection-2022-1.0-py39
18c18
<   - bluesky-kafka >=0.9.0  # [not win]
---
>   - bluesky-kafka >=0.8.0  # [not win]
21d20
<   - bluesky-queueserver-api
29c28
<   - databroker >=1.2.5,<2
---
>   - databroker >=1.2.0
76c75
<   - nslsii >=0.4.0
---
>   - nslsii >=0.3.2
83c82
<   - ophyd >=1.6.4
---
>   - ophyd >=1.6.3
113c112
<   - silx >=0.15.2
---
>   - silx >=0.15.2,<1
```

**Comment by mrakitin** [comment]
> ```diff
> diff 2022-2.0-py39-env.yml ~/dama/conda-pack-template/envs/nsls2-collection-2022-1.0-py39-env.yml
> 1c1
> < name: 2022-2.0-py39
> ---
> > name: nsls2-collection-2022-1.0-py39
> 18c18
> <   - bluesky-kafka >=0.9.0  # [not win]
> ---
> >   - bluesky-kafka >=0.8.0  # [not win]
> 21d20
> <   - bluesky-queueserver-api
> 29c28
> <   - databroker >=1.2.5,<2
> ---
> >   - databroker >=1.2.0
> 76c75
> <   - nslsii >=0.4.0
> ---
> >   - nslsii >=0.3.2
> 83c82
> <   - ophyd >=1.6.4
> ---
> >   - ophyd >=1.6.3
> 113c112
> <   - silx >=0.15.2
> ---
> >   - silx >=0.15.2,<1
> ```

**Review by mrakitin — APPROVED** [review]
Looks good to me, thanks, @gwbischof! The environments will appear in the Actions tab.

**Comment by gwbischof** [comment]
```diff 
2022-2.0-py37-env.yml 2022-2.0-py39-env.yml
1c1
< name: 2022-2.0-py37
---
> name: 2022-2.0-py39
10c10
<   - python >=3.7,<3.8
---
>   - python >=3.9,<3.10
```

File: envs/2022-2.0-py37-env.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  - tzlocal !=3.0
+  - xlrd
+  - xlwt
+  # TODO: xmidas has a dependency of scipy>1.7 that is conflicting with some
+  # packages in this env (also conflicts with scipy>1.6). Need to resolve it
+  # first before including of the xmidas package.
+  # - xmidas >=0.1.1
+  - xpdan >=0.8.2
+  - xray-vision >=0.1.1
+  - xraylarch  >=0.9.58
```
Comment (mrakitin) [inline envs/2022-2.0-py37-env.yml]:
```suggestion
  - xraylarch >=0.9.58
```

File: envs/2022-2.0-py39-env.yml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  - tzlocal !=3.0
+  - xlrd
+  - xlwt
+  # TODO: xmidas has a dependency of scipy>1.7 that is conflicting with some
+  # packages in this env (also conflicts with scipy>1.6). Need to resolve it
+  # first before including of the xmidas package.
+  # - xmidas >=0.1.1
+  - xpdan >=0.8.2
+  - xray-vision >=0.1.1
+  - xraylarch  >=0.9.58
```
Comment (mrakitin) [inline envs/2022-2.0-py39-env.yml]:
```suggestion
  - xraylarch >=0.9.58
```

**Review by mrakitin — COMMENTED** [review]
Minor style suggestions.

**Review by mrakitin — APPROVED** [review]
Looks good to me now, I don't think it needs any changes as long as the CI builds pass.

**Comment by mrakitin** [comment]
Published as https://zenodo.org/record/6462525.

**Pull request merged**